governance/api.go

```diff
@@ -165,6 +165,18 @@ func (api *PublicGovernanceAPI) ItemsAt(num *rpc.BlockNumber) (map[string]interf
 	}
 }
 
+func (api *PublicGovernanceAPI) GetStakingInfo(num *rpc.BlockNumber) *reward.StakingInfo {
```

Comment:
Can I ask why you remove error return value? In other apis, it returns `kerrors.ErrPendingBlockNotSupported` if it is PendingBlockNumber.

Reply:
I removed it since there is no error for `reward.GetStakingInfo`, but adding error is better for consistency. Thanks.